docs/concepts/endpoint.md

```diff
@@ -0,0 +1,83 @@
+---
+id: endpoint
+title: Endpoint
+---
+
+---
+id: client
+title: Client
+---
```

Comment:
Why is there a client header on the endpoint.md file? Is it a mistake?

Reply:
yea it is